src/lib.rs

```diff
@@ -69,26 +74,31 @@ compile_error!(
     Please select one of the following
 
     (Note: `x` denotes any character in [a-z])
-    *   stm32f301
-    *   stm32f318
-    *   stm32f302xb
-    *   stm32f302xc
-    *   stm32f302xd
-    *   stm32f302xe
-    *   stm32f302x6
-    *   stm32f302x8
-    *   stm32f303xb
-    *   stm32f303xc
-    *   stm32f303xd
-    *   stm32f303xe
-    *   stm32f303x6
-    *   stm32f303x8
-    *   stm32f373
-    *   stm32f378
-    *   stm32f334
-    *   stm32f328
-    *   stm32f358
-    *   stm32f398
+    * stm32f301x6
```

Comment:
This list gets longer and longer. I think we should reduce this error message to the essential minimum and point to the documentation, what devices / features are supported.

Reply:
Note: The `TODO` in line 66 is no longer true. I tripped up on my own TODO note. If you like to, you can remove it :)

Reply:
Maybe this message can be compacted without dropping the convenience of listing all supported devices by putting multiple names on a single line?  For example:

```rust
compile_error!(
    "This crate requires you to specify your target chip as a feature.

    Please select one of the following (`x` denotes any character in [a-z]):

        stm32f301x6 stm32f301x8
        stm32f318x8
        stm32f302x6 stm32f302x8 stm32f302xb stm32f302xc stm32f302xd stm32f302xe
        stm32f303x6 stm32f303x8 stm32f303xb stm32f303xc stm32f303xd stm32f303xe
        stm32f328x8
        stm32f358xc
        stm32f398xe
        stm32f373x8 stm32f373xb stm32f373xc
        stm32f378xc
        stm32f334x4 stm32f334x6 stm32f334x8

    Example: The STM32F3Discovery board has a STM32F303VCT6 chip.
    So you want to expand your call to `cargo` with `--features stm32f303xc`.
    For more information, see README -> Selecting the right chip.
    "
);
```

Reply:
That would be a good alternative. I'm fine with both.